### ISSUE TITLE
Use FastCGI server

### ISSUE THREAD

**Issue by nubs** [facebookarchive/heroku-buildpack-hhvm#12]
From what I understand, the builtin server is being deprecated - is that correct?  If so, this buildpack should be updated to use FastCGI, with nginx/apache/? as the server.

**Comment by ptarjan** [comment]
Yup! We'll be doing this with the upcoming 3.0.0 release.

**Comment by ClaudiuC** [comment]
@ptarjan How are you thinking to decide handling nginx/apache - which do we choose?

**Comment by ptarjan** [comment]
@ClaudiuC whatever the PHP buildpack chooses I'm hoping to be identical to them.

**Comment by ClaudiuC** [comment]
@ptarjan Makes sense :+1: . Looks like https://github.com/CHH/heroku-buildpack-php uses nginx. I think I could find some time to make the updates this week-end if you don't get a chance, if you could just get a new package up on http://dl.hhvm.com/heroku/

**Comment by ptarjan** [comment]
@ClaudiuC yup! That would be wonderful. Basically I want it to be the same but with our package instead. I'll work on getting the package up for 10.04 up here http://dl.hhvm.com/ubuntu/dists/lucid/

**Comment by jowy** [comment]
Any news on this? :)!

**Comment by JosefJezek** [comment]
Any news?
